src/cascadia/WinRTUtils/inc/WtExeUtils.h

```diff
@@ -19,15 +34,18 @@ _TIL_INLINEPREFIX bool IsDevBuild()
 {
     // use C++11 magic statics to make sure we only do this once.
     static bool isDevBuild = []() -> bool {
-        try
+        if (IsPackaged())

```

Comment:
I thought it would be redundant to access `ApplicationModel` if it doesn't work unpackaged anyways.
(Same below.)